DataManagementSystem/Agent/RequestOperations/MoveReplica.py

```diff
@@ -0,0 +1,447 @@
+""" MoveReplica operation handler
+
+    This handler moves replicas from source SEs to target SEs. Replicas are first replicated to target SEs and then removed from the source SEs
+"""
+
+__RCSID__ = "$Id $"
+
+# # imports
+import re, os, string
+# # from DIRAC
+from DIRAC import S_OK, S_ERROR, gLogger
+from DIRAC.FrameworkSystem.Client.MonitoringClient import gMonitor
+from DIRAC.Core.Utilities.Adler import compareAdler
+
+from DIRAC.DataManagementSystem.Client.DataManager                                import DataManager
+from DIRAC.DataManagementSystem.Agent.RequestOperations.DMSRequestOperationsBase  import DMSRequestOperationsBase
+
+from DIRAC.Resources.Storage.StorageElement                                       import StorageElement
+from DIRAC.Resources.Catalog.FileCatalog                                          import FileCatalog
+
+def filterReplicas( opFile, logger = None, dataManager = None ):
```

Comment:
Yes, exactly. I agree... For the reuse of filterReplicas and other things taken from ReplicateAndRegister, would it make sense to create a DataManagement utility to be used from both ReplicateAndRegister and MoveFile?